eslint.config.ts

```diff
@@ -205,6 +204,7 @@ const config: ReturnType<typeof tseslint.config> = tseslint.config(
     files: ['src/**/*.ts'],
     rules: {
       'jsdoc/require-jsdoc': 'error',
+      'no-restricted-globals': ['error', 'Intl', 'Buffer'],
```

Comment:
We could extend this to all globals that aren't available in the browser.

````ts
import globals from 'globals';

const browserGlobals = new Set(Object.keys(globals.browser));
const disallowedGlobals = Object.keys(globals.node).filter(
  (e) => !browserGlobals.has(e)
);
````

```suggestion
      'no-restricted-globals': ['error', 'Intl', ...disallowedGlobals],
```

Reply:
This looks promising. I would argue that the `Set` is not required since there cannot be duplicated object keys.  Instead of `browserGlobals.has(e)` you could then use `browserGlobals.includes(e)`.

---

Would this have drastic decreases in performace for the lint step? 🤔

Reply:
> Would this have drastic decreases in performace for the lint step? 🤔

I haven't tested that yet, but I kind of don't like it because I have to add a new dependency for that.
It works for the cypress tests out of the box, not sure which property is responsible for that.